### ISSUE TITLE
Automatic upload of pictures taken from phone

### ISSUE THREAD

**Issue by moy** [Piwigo/piwigo-flutter-app#12]
One killer-feature of Google Photos is the [automatic back-up](https://support.google.com/photos/answer/6193313?co=GENIE.Platform%3DAndroid&hl=en&oco=1). Once set up, pictures taken from the phone are automatically uploaded to the cloud storage, helpful both for easy sharing (e.g. take a picture from your phone, share from your computer's web browser) and backup (pictures are still there if you break your phone).

It would be awesome to have this in Piwigo's App.

**Comment by SimonMGS** [comment]
Yes, this would be a very valuable feature, IMO. Perhaps just auto-uploading all photos to a private "Phone" album, to be sorted out later.

**Comment by Pascaloub** [comment]
I also think that this would be a great feature. I use this of kind of tools when I need to quickly share pictures with my colleges for use on social network when I'm on a report. I download photo from the camera to my phone and send them from there. It would be great to be able to automatically retrieve them on piwigo.
If I may, I would suggest you take a look to the android nextcloud application, they have a very good tool for this with all the features we could dream of on piwigo ng. (paramètres/automatic sending)
thx for your work,

Pascal

**Comment by SimonStiil** [comment]
+1
